src/libcore/mem.rs

```diff
@@ -180,6 +180,31 @@ pub fn forget<T>(t: T) {
 /// More specifically, this is the offset in bytes between successive
 /// items of the same type, including alignment padding.
```

Comment:
While you're here, consider mentioning that this is "successive elements in an array with that item type", since layout optimization can (in theory at least) change the layout of "successive items of the same type" in a struct.

Reply:
Done.